inifix/format.py

```diff
@@ -2,44 +2,86 @@
 import os
 import re
 import sys
+import warnings
 from typing import Optional
 from typing import Sequence
 
 from inifix.io import load
 
 NAME_COL_MINSIZE = 10
 COMMENT_COL = 80
+DEFAULT_PADDING_SIZE = 4
```

Comment:
Well, all of them are "default" now, aren't they ?
```suggestion
PADDING_SIZE = 4
```